Solutions/Salem Virtual Cyber Analyst/Playbooks/SendAlertToSalem/azuredeploy.json

```diff
@@ -40,8 +40,8 @@
         }
     },
     "variables": {
-        "SalemEventHubConnectionName": "[concat('SalemEventHub-', parameters('PlaybookName'))]",
-        "SentinelConnectionName": "[concat('MicrosoftSentinel-', parameters('PlaybookName'))]"
+        "SalemEventHubConnectionName": "Salem-EventHub",
+        "SentinelConnectionName": "Salem-MicrosoftSentinel"
```

Comment:
Updating Connection Names to be non-dynamic.  This will help with connection reusability if other Salem playbooks are deployed later